R/assertions.r

```diff
@@ -141,7 +142,7 @@ on_failure(is.date) <- function(call, env) {
 #' see_if(mean %has_args% "y")
 has_args <- function(f, args, exact = FALSE) {
   assert_that(is.function(f))
-  
+
```

Comment:
Sorry, I have my RStudio set to trim trailing whitespace when I save a file. Let me know if you'd like me to revert all of these